.s2i/bin/run

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+echo "omp s2i"
+
+sed -i 's@v0.0.1-local@'"$OMP_FRONTEND_GIT_TAG"'@' manifest.json
```

Comment:
Should this be `public/manifest.json`?

Reply:
no. that file is found at $HOME after the build. This gets executed when the container is launched

```
sh-4.2$ cd
sh-4.2$ pwd
/opt/app-root/src
sh-4.2$ ls -l
total 76
-rw-rw-r--. 1 default    root  1531 May  1 20:48 asset-manifest.json
drwxrwxr-x. 2 default    root    33 May  1 20:48 config
-rw-rw-r--. 1 default    root  2316 May  1 20:48 index.html
-rw-rw-r--. 1 1004860000 root   383 May  1 20:52 manifest.json
-rw-rw-r--. 1 default    root 38190 May  1 20:48 oil_logo.png
-rw-rw-r--. 1 default    root 13010 May  1 20:48 precache-manifest.5074d121ecad3fb8267489d231f074d4.js
-rw-rw-r--. 1 default    root    57 May  1 20:48 robots.txt
-rw-rw-r--. 1 default    root  1181 May  1 20:48 service-worker.js
drwxrwxr-x. 5 default    root    40 May  1 20:48 static
```

Reply:
Got it 👍